kiauh/components/klipper_firmware/menus/klipper_flash_help_menu.py

```diff
@@ -143,6 +143,8 @@ def print_menu(self) -> None:
         count = 62 - len(color) - len(RESET_FORMAT)
         subheader1 = f"{COLOR_CYAN}USB:{RESET_FORMAT}"
         subheader2 = f"{COLOR_CYAN}UART:{RESET_FORMAT}"
+        subheader3 = f"{COLOR_CYAN}USB DFU:{RESET_FORMAT}"
+        subheader4 = f"{COLOR_CYAN}USB RP2 Boot:{RESET_FORMAT}"
```

Comment:
We should name it identically to the option that is selectable. So either rename the method to RP2 Boot or rename this heading to RP2040?